### PR TITLE
64 use case like story

### PR DESCRIPTION
**Pull request by Serdiuk-Andrii**

### CODE REVIEW

**Review by sarkar50-lab — DISMISSED** [review]

File: src/main/java/usecases/like_story/LsGatewayInputData.java [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    private final int storyId;
+
+    /**
+     * @param storyId The id of the story that should be liked
+     * */
+    public LsGatewayInputData(int storyId) {
+        this.storyId = storyId;
+    }
+
+
```
Comment (mariana0412) [inline src/main/java/usecases/like_story/LsGatewayInputData.java:14]: two empty lines

File: src/test/java/usecases/like_story/LsTests.java [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+                return 0;
+            }
+            lock.unlock();
+            return result;
+        }
+
+    }
+
+
+}
```
Comment (mariana0412) [inline src/test/java/usecases/like_story/LsTests.java:124]: two empty lines

**Review by mariana0412 — DISMISSED** [review]
Good job!

**Review by mariana0412 — APPROVED** [review]

**Review by AlekseyPanas — APPROVED** [review]

File: src/test/java/usecases/like_story/LsTests.java [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+    private Random random;
+
+    @Before
+    public void setup() {
+        presenter = new TestLsPresenter();
+        repository = new TestLsGateway();
+        interactor = new LsInteractor(presenter, repository);
+        random = new Random();
+    }
+
```
Comment (bac423) [inline src/test/java/usecases/like_story/LsTests.java:30]: Please, add test description

File: src/test/java/usecases/like_story/LsTests.java [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+                Thread.onSpinWait();
+            }
+            Assertions.assertEquals(currentNumberOfResponses + numberOfLikes,
+                    presenter.responses.size());
+                    }
+        for (LsOutputData data : presenter.responses) {
+            Assertions.assertEquals(data.getResponse().getCode(), Response.ResCode.SUCCESS);
+        }
+    }
+
```
Comment (bac423) [inline src/test/java/usecases/like_story/LsTests.java:56]: Please, add test description

**Review by sarkar50-lab — DISMISSED** [review]
Good job!

**Review by bac423 — APPROVED** [review]
Amazing job, fix doc

**Review by sarkar50-lab — APPROVED** [review]

**Review by bac423 — APPROVED** [review]
Great job

**Review by AlekseyPanas — APPROVED** [review]

**Pull request merged**